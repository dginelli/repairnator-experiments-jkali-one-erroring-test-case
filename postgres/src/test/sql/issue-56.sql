select parent_id, count(*) from so_posts where zdb('so_posts', ctid) ==> '#expand<parent_id=<this.index>parent_id>(beer)' group by 1 order by 1 desc;