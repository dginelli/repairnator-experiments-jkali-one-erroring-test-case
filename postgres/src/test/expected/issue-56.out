select parent_id, count(*) from so_posts where zdb('so_posts', ctid) ==> '#expand<parent_id=<this.index>parent_id>(beer)' group by 1 order by 1 desc;
 parent_id | count 
-----------+-------
           |    21
    270821 |     2
    267981 |     8
    263389 |     1
    258311 |     5
    256833 |     8
    250833 |    10
    244750 |    10
    237245 |    10
    234251 |     6
    231925 |     2
    220676 |     2
    219101 |     4
    215682 |     1
    211682 |    13
    211608 |    10
    211451 |     3
    210002 |     6
    206668 |     8
    204286 |     2
    203267 |     1
    198145 |     5
    197162 |     4
    197133 |     2
    186127 |     1
    177859 |     5
    166959 |     2
    165543 |     5
    164429 |     7
    159754 |    11
    156290 |     7
    155488 |    19
    153613 |     6
    152733 |     8
    151229 |     7
    148758 |     2
    145784 |    10
    144656 |     4
    142028 |     3
    141329 |    15
    140876 |     7
    140051 |     6
    139111 |     3
    135499 |     2
    134687 |     3
    133768 |    16
    130820 |     9
    128428 |     6
    121528 |     9
    120560 |    41
    119060 |     4
    118690 |     2
    117638 |    14
    116185 |    17
    112730 |     2
    108785 |     3
    108552 |     8
    106980 |   137
    106935 |    11
    105952 |     3
    105851 |     8
    105089 |    11
    104346 |     8
    103567 |     9
    102894 |     3
    102873 |     3
    102689 |    16
    101988 |    17
    100903 |    14
    100685 |    20
     98833 |    20
     98340 |     5
     93711 |     6
     92862 |    34
     91683 |    18
     87792 |     6
     87563 |     1
     85864 |     6
     82632 |     9
     81743 |    18
     80988 |     8
     79419 |     7
     79134 |     8
     78477 |    11
     78291 |     4
     77662 |     4
     76415 |     8
     74249 |     1
     74235 |     6
     73863 |     7
     72937 |     8
     71926 |     2
     71486 |     4
     70445 |     6
     68954 |     4
     68740 |    29
     66708 |    13
     62418 |     6
     61764 |    10
     61392 |     8
     58779 |     4
     58768 |     8
     58086 |     7
     56667 |     2
     53788 |     7
     53648 |     7
     53231 |     4
     52816 |     8
     52534 |     8
     51913 |    13
     50299 |    14
     49018 |     7
     46084 |    22
     44936 |     5
     44177 |   360
     43356 |     4
     41514 |    20
     40593 |     4
     39720 |    11
     36612 |    41
     34584 |    12
     29852 |    14
     27862 |     3
     27091 |     6
     26771 |    17
     26609 |     9
     26349 |     6
     25267 |     2
     24361 |     4
     24159 |     6
     21575 |    17
     16301 |    27
     15759 |    16
     14720 |    21
     12343 |     5
     12070 |    10
     11813 |     4
     11565 |    19
     10574 |     7
      8301 |    19
      7126 |    11
      5384 |    60
      3233 |    64
(143 rows)

